documentation/assemblies/metrics/assembly-metrics.adoc

```diff
@@ -11,6 +11,9 @@ You can use Prometheus and Grafana to monitor your Strimzi deployment.
 You can monitor your Strimzi deployment by viewing key metrics on dashboards and setting up alerts that trigger under certain conditions.
 Metrics are available for each of the components of Strimzi.
 
+Some metrics have to be explicitly enabled to be collected. For example, Kafka CR 'spec.kafka.listeners.authentication' and 'spec.kafka.authorization' have 'enableMetrics' property which needs to be set to 'true' when using 'oauth' authentication or 'keycloak' authorization for corresponding metrics to be collected.
+Similarly for client authentication in KafkaBridge, KafkaConnect, KafkaMirrorMaker, or KafkaMirrorMaker2 CRs when 'oauth' type is used.
```

Comment:
I think this is misleading as it suggests this applies in general. While in reality, it applies only to `oauth` authentication and `opa` authorization / `keycloak` authentication. So I think it needs to be approached from the other side ... i.e. _if you are using one of these, you will need to explicitly enable the metrics ..._

Reply:
Ok, how about:

```suggestion
If you are using `oauth` authentication or `opa` / `keycloak` authorization the collection of their specific metrics has to be explicitly enabled. For example, Kafka CR 'spec.kafka.listeners.authentication' and 'spec.kafka.authorization' have 'enableMetrics' property which needs to be set to 'true'.
Similarly for client authentication in KafkaBridge, KafkaConnect, KafkaMirrorMaker, or KafkaMirrorMaker2 CRs when 'oauth' type is used.
```

Reply:
Yeah, I think this sounds good to me. It makes it more clear who should worry about this.